keyboards/ashwing66/config.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define ENCODER_MAP_KEY_DELAY 10
+#define ENCODER_DEFAULT_POS 0x3
```

Comment:
Is this actually needed for correct behavior?

Reply:
To allow for potential differences with encoders used. Does not appear to change behaviour from own testing with components, cant say for other EC11 encoders used.